Moving more common browser functions to shared module

mtlynch opened the pull request:
This moves some small functions from the HTML5 driver into a common module
for browser functions so that other browser drivers can use the same
functionality. Also moves error messages that will be common across browser
drivers to the common module.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/m-lab/ndt-e2e-clientworker/42)
<!-- Reviewable:end -->

yachang commented:
Review status: 0 of 4 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[tests/test_browser_client_common.py, line 89 \[r1\]](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42#-KGNkJn1HZVEL4ZaQZWl:-KGNkJn1HZVEL4ZaQZWm:2023523081) ([raw file](https://github.com/m-lab/ndt-e2e-clientworker/blob/97df46bde629f2edee59a79f0c3e3040a9e20c31/tests/test_browser_client_common.py#L89)):*
20 is 20 seconds?

---

*[tests/test_browser_client_common.py, line 121 \[r1\]](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42#-KGNq58bsMaTt8P4ZqXs:-KGNq58bsMaTt8P4ZqXt:1331883382) ([raw file](https://github.com/m-lab/ndt-e2e-clientworker/blob/97df46bde629f2edee59a79f0c3e3040a9e20c31/tests/test_browser_client_common.py#L121)):*
What will happen if there are multiple elements with that text?

---

*[tests/test_html5_driver.py, line 64 \[r1\]](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42#-KGNokcWlIW_IVtBgxEX:-KGNokcWlIW_IVtBgxEY:-2083941063) ([raw file](https://github.com/m-lab/ndt-e2e-clientworker/blob/97df46bde629f2edee59a79f0c3e3040a9e20c31/tests/test_html5_driver.py#L64)):*
Why do you need driver as param? only "text" needed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42)*
<!-- Sent from Reviewable.io -->

mtlynch commented:
Review status: 0 of 4 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[tests/test_browser_client_common.py, line 89 \[r1\]](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42#-KGNkJn1HZVEL4ZaQZWl:-KGNrSuhvKitnQLvPbpe:-1123495366) ([raw file](https://github.com/m-lab/ndt-e2e-clientworker/blob/97df46bde629f2edee59a79f0c3e3040a9e20c31/tests/test_browser_client_common.py#L89)):*
Yes, 20 seconds, but we're mocking out the wait, so this happens instantaneously rather than taking 20 seconds.

---

*[tests/test_browser_client_common.py, line 121 \[r1\]](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42#-KGNq58bsMaTt8P4ZqXs:-KGNrVNa9gfilqQT1Iix:-978139078) ([raw file](https://github.com/m-lab/ndt-e2e-clientworker/blob/97df46bde629f2edee59a79f0c3e3040a9e20c31/tests/test_browser_client_common.py#L121)):*
It will return the first one that matches. I've updated the docstring for `find_element_containing_text` to clarify.

---

*[tests/test_html5_driver.py, line 64 \[r1\]](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42#-KGNokcWlIW_IVtBgxEX:-KGO1NFqQejusou6C25y:-1470888771) ([raw file](https://github.com/m-lab/ndt-e2e-clientworker/blob/97df46bde629f2edee59a79f0c3e3040a9e20c31/tests/test_html5_driver.py#L64)):*
It has to match the function signature of `find_elements_by_text` but we don't have to name `driver`, so I've replaced it with `_`.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42)*
<!-- Sent from Reviewable.io -->

yachang commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 4 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/ndt-e2e-clientworker/42#-:-KGO3FadoZoikQx_zLtc:980077401)*
<!-- Sent from Reviewable.io -->
